shared_model/interfaces/common_objects/ordering.hpp

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef IROHA_SHARED_MODEL_ORDERING_HPP
+#define IROHA_SHARED_MODEL_ORDERING_HPP
+
+#include "interfaces/common_objects/types.hpp"
+
+namespace shared_model {
+  namespace interface {
+
+    /// Provides ordering data for pagination
+    class Ordering final {
```

Comment:
please add a `std::string toString() const` method.

and please add it to the `toString` of `PaginationMeta`